taskcat/_cli_modules/install.py

```diff
@@ -0,0 +1,171 @@
+# pylint: disable=duplicate-code
+import logging
+from io import BytesIO
+from pathlib import Path
+from time import sleep
+
+from dulwich import porcelain
+from dulwich.config import ConfigFile, parse_submodules
+
+from taskcat._cfn.stack import Tag
+from taskcat._cfn.threaded import Stacker
+from taskcat._client_factory import Boto3Cache
+from taskcat._config import Config
+from taskcat._name_generator import generate_name
+from taskcat._s3_stage import stage_in_s3
+from taskcat.exceptions import TaskCatException
+
+LOG = logging.getLogger(__name__)
+
+
+class Install:
+    """[ALPHA] installs a stack into an AWS account/region"""
+
+    PKG_CACHE_PATH = Path("~/.taskcat_package_cache/").expanduser().resolve()
+
+    # pylint: disable=too-many-branches,too-many-locals
+    def __init__(  # noqa: C901
+        self,
+        package: str,
+        aws_profile: str = "default",
+        region="default",
+        parameters="",
+        name="",
+        wait=False,
+    ):
+        """
+        :param package: name of package to install can be a path to a local package,
+        a github org/repo, or an AWS Quick Start name
+        :param aws_profile: aws profile to use for installation
+        :param region: regions to install into, default will use aws cli configured
+        default
+        :param parameters: parameters to pass to the stack, in the format
+        Key=Value,AnotherKey=AnotherValue or providing a path to a json or yaml file
+        containing the parameters
+        :param name: stack name to use, if not specified one will be automatically
+        generated
+        :param wait: if enabled, taskcat will wait for stack to complete before exiting
+        """
+        boto3_cache = Boto3Cache()
+        if not name:
+            name = generate_name()
+        if region == "default":
+            region = boto3_cache.get_default_region(profile_name=aws_profile)
+        path = Path(package).resolve()
+        if Path(package).resolve().is_dir():
+            package_type = "local"
+        elif "/" in package:
+            package_type = "github"
+        else:  # assuming it's an AWS Quick Start
```

Comment:
I also thought we were getting away from aws-quickstart assumptions.

Reply:
this doesn't enforce anything quickstart specific, it just uses quickstart org as the default "repository" for installation. So install that can be done simply from any github repo:

`taskcat install myorg/myrepo`